walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +122,66 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void optionalAttendeeIsNotConsidered() {
```

Comment:
This testcase and the next are very similar, and it's hard for a reader to understand why optional attendee would be considered there but not here.
Prefer to include more context in the testcase name (i.e., *why* attendee C is not considered here).